content/intro-to-storybook/vue/es/simple-component.md

```diff
@@ -0,0 +1,279 @@
+
+- `title` – un string que describe la tarea
+- `state` - ¿en qué lista se encuentra la tarea actualmente y está marcado el checkbox?
+
+A medida que comencemos a construir `Task`, primero escribiremos nuestras pruebas para los estados que corresponden a los distintos tipos de tareas descritas anteriormente. Luego, utilizamos Storybook para construir el componente de forma aislada usando datos de prueba. Vamos a “testear visualmente” la apariencia del componente a medida que cambiemos cada estado.
+
+Este proceso es similar a [Test-driven development](https://en.wikipedia.org/wiki/Test-driven_development) (TDD) al que podemos llamar “[Visual TDD](https://blog.hichroma.com/visual-test-driven-development-aec1c98bed87)”.
+
+## Ajustes iniciales
+
+Primero, vamos a crear el componente Task y el archivo de historias de Storybook que lo acompaña: `src/components/Task.js` y `src/components/Task.stories.js`.
+
+Comenzaremos con una implementación básica de `Task`, simplemente teniendo en cuenta los atributos que sabemos que necesitaremos y las dos acciones que puedes realizar con una tarea (para moverla entre las listas):
+
+```html
+<!--src/components/Task.vue-->
+<template>
+  <div class="list-item">
+    <input type="text" :readonly="true" :value="this.task.title" />
+  </div>
+</template>
+
+<script>
+  export default {
+    name: 'task',
+    props: {
+      task: {
+        type: Object,
+        required: true,
+        default: () => ({}),
+      },
+    },
+  };
+</script>
+```
+
+Arriba, renderizamos directamente `Task` basándonos en la estructura HTML existente de la app Todos.
+
+A continuación creamos los tres estados de prueba de Task dentro del archivo de historia:
+
+```javascript
+//src/components/Task.stories.js
+import { action } from '@storybook/addon-actions';
+import Task from './Task';
+export default {
+  title: 'Task',
+  // Our exports that end in "Data" are not stories.
+  excludeStories: /.*Data$/,
+};
+export const actionsData = {
+  onPinTask: action('onPinTask'),
+  onArchiveTask: action('onArchiveTask'),
+};
+
+export const taskData = {
+  id: '1',
+  title: 'Test Task',
+  state: 'Task_INBOX',
+  updated_at: new Date(2019, 0, 1, 9, 0),
+};
+
+const taskTemplate = `<task :task="task" @archiveTask="onArchiveTask" @pinTask="onPinTask"/>`;
+
+// default task state
+export const Default = () => ({
+  components: { Task },
+  template: taskTemplate,
+  props: {
+    task: {
+      default: () => (taskData)
+    },
+  },
+  methods: actionsData,
+});
+// pinned task state
+export const Pinned = () => ({
+  components: { Task },
+  template: taskTemplate,
+  props: {
+    task: {
+      default: {
+        ...taskData,
+        state: 'TASK_PINNED',
+      },
+    },
```

Comment:
@IvanSotelo this slipped my mind, the snippet needs to be updated aswell to a factory function or the issue with jest/vue cli will present itself. 
It should be 
```javascript
task: {
      default: () => ({
        ...taskData,
        state: "TASK_PINNED"
      })
    }
````

And the same logic for the archived state below